ResearchKit/Common/ORKTaskViewController.m

```diff
@@ -1021,6 +1036,11 @@ - (ORKStepViewController *)viewControllerForStep:(ORKStep *)step {
         
         stepViewController.restorationIdentifier = step.identifier;
         stepViewController.restorationClass = stepViewControllerClass;
+        if ([step isKindOfClass:[ORKReviewStep class]] && [stepViewController isKindOfClass:[ORKReviewStepViewController class]]) {
+            ORKReviewStepViewController *reviewStepViewController = (ORKReviewStepViewController *) stepViewController;
+            reviewStepViewController.resultSource = self.result;
```

Comment:
Aren't supposed to only supply `resultSource` when ORKReviewStep is not carrying one?